src/features/analytics/config.ts

```diff
@@ -0,0 +1,16 @@
+export const PUBLIC_POSTHOG_HOST = 'https://eu.posthog.com';
+
+export const PRODUCTION_TRACKING_MODE_ENABLED =
+  process.env.PRODUCTION_MODE_TRACKING === 'true';
+
+export const PRODUCTION_NETWORK_ID_TO_POSTHOG_PROJECT_ID_MAP = {
+  ['mainnet']: 6621,
+  ['preprod']: 6620,
+  ['preview']: 6619,
```

Comment:
@renanvalentin we will be using the same Posthog project for all networks (that will be different in Nami as compared to Lace), so the network should be an event property instead